apps/forms/mixins.py

```diff
@@ -43,41 +47,23 @@ def _setup_container_image_helper(self):
     def clean_image(self):
         """Validate the container image input."""
         image = self.cleaned_data.get("image", "").strip()
+
         if not image:
             self.add_error("image", "Container image field cannot be empty.")
             return image
 
-        # Ignore non-Docker images for now
-        if "docker.io" not in image:
-            return image
-
-        # Split image into repository and tag
-        if ":" in image:
-            repository, tag = image.rsplit(":", 1)
-        else:
-            repository, tag = image, "latest"
-
-        repository = repository.replace("docker.io/", "", 1)
-
-        # Ensure repository is in the correct format
-        # The request to Docker hub will fail otherwise
-        if "/" not in repository:
-            repository = f"library/{repository}"
-
-        # Docker Hub API endpoint for checking the image
-        docker_api_url = f"{settings.DOCKER_HUB_TAG_SEARCH}{repository}/tags/{tag}"
+        if "ghcr.io" in image:
+            try:
+                validate_ghcr_image(image)
+            except Exception as e:
```

Comment:
```suggestion
            except ValidationError as e:
```

Why don't you catch it like this?

Reply:
Yes, good catch.. will change